src/server/app/resources/services/resources.server.service.js

```diff
@@ -109,11 +91,11 @@ module.exports = function() {
 							'owner._id': user._id
 						}];
 
-						return doSearch(query, sortParams, page, limit);
+						return Resource.pagingSearch(query, sortParams, page, limit);
```

Comment:
Absolutely! I'll add some basic tests around each of these `pageable` models too (instead of the handful that are there on `Messages` now).